model/patch_lifecycle.go

```diff
@@ -446,8 +446,13 @@ func FinalizePatch(ctx context.Context, p *patch.Patch, requester string, github
 		if err = tasksToInsert.InsertUnordered(sessCtx); err != nil {
 			return nil, errors.Wrapf(err, "error inserting tasks for version '%s'", patchVersion.Id)
 		}
+		if p.IsParent() && len(childPatchIds) > 0 {
+			if err = p.SetChildPatches(); err != nil {
```

Comment:
Sorry I didn't ask about this earlier but: these patches are created from running ProcessTriggerAliases so there's not really a reason we need to do this within the transaction / that it's worth passing through just for that. Can we do this before calling ProcessTriggerAliases in the graphql function instead, if childPatchIds length > 0?